test/index.test.js

```diff
@@ -236,6 +236,45 @@ describe('liana > index', () => {
           expect(response.status).not.toStrictEqual(404);
         });
       });
+
+      describe('when implementation exposes RequestUnflattener middleware', () => {
```

Comment:
Let's keep an eye on this because I've done quite the same things in this [PR](https://github.com/ForestAdmin/forest-express/pull/770/files) we might encounter conflicts